examples/jsm/nodes/accessors/ViewDirectionNode.js

```diff
@@ -0,0 +1,30 @@
+import Node, { addNodeClass } from '../core/Node.js';
+import { positionLocal, positionWorld } from './PositionNode.js';
+import { cameraPosition } from './CameraNode.js';
+import { nodeProxy } from '../shadernode/ShaderNode.js';
+
+class ViewDirectionNode extends Node {
+
+	constructor( positionNode ) {
+
+		super( 'vec3' );
+
+		this.positionNode = positionNode;
+
+	}
+
+	setup() {
+
+		return this.positionNode.sub( cameraPosition ).normalize();
```

Comment:
@sunag

In the shaders, we use this pattern consistently:
```glsl
vec4 mvPosition = vec4( transformed, 1.0 );

vViewPosition = - mvPosition.xyz

vec3 viewDir = normalize( vViewPosition );
```
So the view direction is a unit vector _from the vertex toward the camera_.

Just making sure you are aware. :-)

Reply:
I added this pattern through `positionViewDirection` and it is widely used in lighting models. 

https://github.com/mrdoob/three.js/blob/b8b82d96ead5a977107410336046bc082d058b07/examples/jsm/nodes/accessors/PositionNode.js#L54-L58

In some tests that I did `MaterialX` seems uses `viewdirection` in `world space` for this node as default instead of `view space`, and apparently will have `local/model space` to select.

Reply:
Sorry, I think you may have misunderstood my comment...

The comment is only about ensuring the _sign_ of "view direction" is correct, and consistent with historical convention, and how three.js defines it.

In the case of local vs. model space, the two definitions should be consistent with each other.

Reply:
Sorry, I think you may have misunderstood my comment...

The comment is only about ensuring the _sign_ of "view direction" is correct, and consistent with historical convention, and how three.js defines it.

In the case of local vs. world space, the two definitions should be consistent with each other.

Reply:
Thanks for the feedback, is an important point, I'll revert this part for now and check this in other PR.